packages/popover/src/utils/positionUtils.ts

```diff
@@ -217,539 +90,98 @@ export function getElementDocumentPosition(
   };
 }
 
-// Gets top offset, left offset, width and height dimensions for a node
-export function getElementViewportPosition(
-  element: HTMLElement | null,
-  scrollContainer?: HTMLElement | null,
-  isReference?: boolean,
-): ElementPosition {
-  if (!element) {
-    return defaultElementPosition;
+/**
+ * Floating UI supports 12 placements out-of-the-box. {@see https://floating-ui.com/docs/useFloating#placement}.
+ * In addition to these placements, we override the `align` prop when it is set to 'center-horizontal' or
+ * 'center-vertical' to create custom placements {@see https://floating-ui.com/docs/offset#creating-custom-placements}
+ */
+export const getFloatingPlacement = (
```

Comment:
So this function converts our `Align` & `Justify` enums into a placement value that `useFloating` accepts?